layouts/partials/header.html

```diff
@@ -8,9 +8,17 @@
 </div>
 <header class="header">
   <div class="header-wrap">
+    <ul class="nav mt0 mb6">
+      <li class="nav-link row-center-y">
+        <a href="https://cloudgov.statuspage.io/" style="align-self: center">Platform Status</a>
+      </li>
+      <li class="nav-link">
+        <a href="https://login.fr.cloud.gov/login" class="action action-outline inline-block">Log in</a>
```

Comment:
Does this link need to be dynamic? I'm not sure if we have different environments for staging, etc.